DXRando/DeusEx/Classes/Keypad.uc

```diff
@@ -53,4 +66,6 @@ function bool WasHacked()
 defaultproperties
 {
     bCodeKnown=False
+    bToggleLock=False
```

Comment:
is this line needed?
```suggestion
```

Reply:
It is, yes. In fact, the keypad has no effect without that line.

Reply:
which keypad doesn't work without this? seems like most of the keypads in the map files already have bToggleLock=False

![image](https://github.com/user-attachments/assets/da2be921-6f61-406d-883a-ec3a79cb9d86)

Reply:
This one in Cathedral (toward vanilla Gunther) doesn't work without it.
![image](https://github.com/user-attachments/assets/e10b97a9-a8b3-4ab7-bbea-87dfbe7e60de)

Reply:
but it works in vanilla and in the current develop branch

the problem is you're hooking in the HackAction function, our existing code is hooked into RunEvents and ToggleLocks

I don't think we need this new HackAction function?

Reply:
That's where the door gets set to be unlocked. Where would you suggest putting that code?

Reply:
Given that the keypad is bToggleLock=False already, I would recommend investigating why that helps and solving the root cause

![image](https://github.com/user-attachments/assets/0860d4bb-00de-40b0-bf50-b2595a6e0c41)

Reply:
Alright, the problem seems to be that I was testing an old save where it was set to True.